### DISCUSSION TITLE
Maximum number of input papers

### DISCUSSION THREAD

**Discussion by ClementineY** [lifan0127/ai-research-assistant#88] [Q&A]
Hi!

I wonder what's the maximum number of papers that it can process.

Thanks!

**Comment by lifan0127** [comment]
Hi @ClementineY , I need to check but I think it is 10 by default. Now that LLM context windows are so much larger than ~1 year ago, it may be worthwhile to expand it. Let me think about it 😃

**Reply by TheMetrifiedMe** [reply]
uh, so Aria is unable to summarize like 400 papers based on title and abstract? I might have missed the point of the tool then...

**Reply by lifan0127** [reply]
Hi, this should be possible based on the latest LLMs! I think I am going to make it adjustable so users can make adjustment as needed.
